src/test/java/com/wikia/webdriver/common/core/XMLReader.java

```diff
@@ -21,13 +20,15 @@ private XMLReader() {
    */
   public static String getValue(File file, String key) {
 
+    if (!file.exists() || file.isDirectory()) {
+      throw new ConfigurationRuntimeException("Cannot find a file with credentials");;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)